### PR TITLE
Finishing Lava and adding Liquid Base Class

### PR DESCRIPTION
**Pull request by Petermarcu**
Also added Ice in here.

### CODE REVIEW

File: Decent.Minecraft.Client/WaterLevel.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     /// <summary>
     /// Species of wood.
```
Comment (bleroy) [inline Decent.Minecraft.Client/WaterLevel.cs:4]: I suppose we should update that doc comment :)

File: Decent.Minecraft.Client/WaterLevel.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
     /// </summary>
-    public enum WaterLevel : byte
+    public enum Level : byte
     {
         Source = 0,
```
Comment (bleroy) [inline Decent.Minecraft.Client/WaterLevel.cs:8]: We should probably add a doc comment for the `Source` member...

**Comment by bleroy** [comment]
My comments are not specific to the changeset, so merging...

File: Decent.Minecraft.Client/WaterLevel.cs (hunk 2 above)
Comment (Petermarcu) [reply Decent.Minecraft.Client/WaterLevel.cs:8]: Yeah, I'll do that tonight. I spent time last night trying to dogfood the scripts we have to set up a CI.

Comment (bleroy) [reply Decent.Minecraft.Client/WaterLevel.cs:8]: I can do it too :)

**Pull request merged**